cs251tk/student/markdownify/markdownify.py

```diff
@@ -4,15 +4,24 @@
 from collections import OrderedDict
 from .process_file import process_file
 from .find_warnings import find_warnings
+from ...common import chdir
+from cs251tk.common import run
```

Comment:
```suggestion
from ...common import run, chdir
```

I think we can just import both of these from the same relative import.